docs/tutorials/first_steps/07_building_scripts.rst

```diff
@@ -133,3 +133,31 @@ You can inspect the code in the file viewer in your JupyterLab session or RStudi
 .. code-block:: console
 
     $ renku save -m 'added filter script'
+
+
+Organizing scripts within your project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For this tutorial we have organized our scripts such that they live in the
+``src`` folder. If your project grows to requiring several programming languages
+or bash scripts, you may want to organize them into separate folders as we have
+demonstrated in the example below.
```

Comment:
This is a good way to handle it! Keep the tutorial simple, but explain how to do it for more advanced cases as well.